src/main/java/org/jenkinsci/plugins/durabletask/BourneShellScript.java

```diff
@@ -188,26 +178,33 @@ public String getScript() {
         }
         cmd = cmd.replace("$", "$$"); // escape against EnvVars jobEnv in LocalLauncher.launch
 
-        args.addAll(Arrays.asList("sh", "-c", cmd));
+        if (LAUNCH_DIAGNOSTICS) {
+            args.addAll(Arrays.asList("sh", "-c", cmd));
+        } else {
+            // setsid --fork also works but not in, say, busybox; or even Ubuntu prior to Cosmic
+            args.addAll(Arrays.asList("sh", "-c", "(" + cmd + ") &"));
```

Comment:
[Discussion](https://superuser.com/a/172476/129406) and [more](https://unix.stackexchange.com/a/352923/26736).